Tests/NumLib/TestExtrapolation.cpp

```diff
@@ -195,8 +196,8 @@ void extrapolate(TestProcess const& pcs, IntegrationPointValuesMethod method,
 {
     namespace LinAlg = MathLib::LinAlg;
 
-    auto const tolerance_dx  = 20.0 * std::numeric_limits<double>::epsilon();
-    auto const tolerance_res =  5.0 * std::numeric_limits<double>::epsilon();
+    auto const tolerance_dx  = 30.0 * std::numeric_limits<double>::epsilon();
+    auto const tolerance_res = 15.0 * std::numeric_limits<double>::epsilon();
```

Comment:
Why the relaxed tolerances? (Just curious why this changed, whether it is 5*1e-16 or 15*1e-16 does not really matter).